css-view-transitions-1/Overview.bs

```diff
@@ -1314,6 +1314,8 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. Let |usedTransitionNames| be a new [=/set=] of strings.
 
+		1. Let |captureElements| be a new [=/list=] of elements.
+
 		1. Let |document| be |transition|'s [=relevant global object's=] [=associated document=].
 
 		1. Set |transition|'s [=ViewTransition/initial snapshot containing block size=] to the [=snapshot containing block size=].
```

Comment:
While we're here, can you add a note in the loop below. Something like:

```
We iterate in paint order (back to front) to ensure that this order is cached in |namedElements|.
This defines the DOM order for ::view-transition-group pseudo-elements, such that the element at the back of the paint stack is the first child of ::view-transition.
```

Reply:
Done.